opencti-platform/opencti-graphql/src/modules/ingestion/ingestion-csv-domain.ts

```diff
@@ -185,3 +190,30 @@ export const testCsvIngestionMapping = async (context: AuthContext, user: AuthUs
     nbEntities: allObjects.filter((object: StixObject) => object.type !== 'relationship').length,
   };
 };
+
+export const csvFeedAddInputFromImport = async (context: AuthContext, user: AuthUser, file: Promise<FileHandle>) => {
+  if (!isFeatureEnabled(CSV_FEED_FEATURE_FLAG)) {
+    throw new Error(`${CSV_FEED_FEATURE_FLAG} feature is disabled`);
+  }
+  const parsedData = await extractContentFrom(file);
+
+  // check platform version compatibility
+  if (!isCompatibleVersionWithMinimal(parsedData.openCTI_version, MINIMAL_CSV_FEED_COMPATIBLE_VERSION)) {
+    throw FunctionalError(
+      `Invalid version of the platform. Please upgrade your OpenCTI. Minimal version required: ${MINIMAL_CSV_FEED_COMPATIBLE_VERSION}`,
+      { reason: parsedData.openCTI_version },
+    );
+  }
+
+  return {
+    ...parsedData.configuration,
+    csvMapper: transformCsvMapperConfig(parsedData.configuration.csv_mapper.configuration, context, user),
+  };
+};
+
+export const csvFeedGetCsvMapper = (ingestionCsv: BasicStoreEntityIngestionCsv, context: AuthContext) => {
```

Comment:
Not a blocker but we usually start methods with context parameter. Can you switch parameters ?